### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/python-nox ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

**Comment by monkeywithacupcake** [comment]
Actually, I did not update the badge. I'm working on a new update for .RST files. Please hold on this PR.

**Comment by monkeywithacupcake** [comment]
I updated the Readme and you should see your Open Collective images

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!-- Love nox? Please consider supporting our collective:
+👉  https://opencollective.com/python-nox/donate -->
```
Comment (stsewd) [inline .github/ISSUE_TEMPLATE.md]: We use the multiple issue template format

https://github.com/theacodes/nox/tree/master/.github/ISSUE_TEMPLATE

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,3 +53,5 @@ If you've contributed to Nox, you can get a cute little Nox sticker. Reach out t
 ## Getting paid
 
 Contributions to Nox can be expensed through [our Open Collective](https://opencollective.com/python-nox). The maintainers will let you know when and for how much you can expense contributions, but always feel free to ask. 
+
+
```
Comment (stsewd) [inline CONTRIBUTING.md]: Extra new lines

File: README.rst [hunk 3]
```diff
@@ -9,7 +9,37 @@ The latest documentation is available on `Read the Docs`_.
 
 The source code is available on `GitHub`_.
 
+Contributors
++++++++
```
Comment (stsewd) [inline README.rst]: All headers should match the same number of characters as the title

File: README.rst [hunk 4]
```diff
@@ -9,7 +9,37 @@ The latest documentation is available on `Read the Docs`_.
...
 
+Code Contributors
+------------------
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```
Comment (stsewd) [inline README.rst]: This should be a rst format link

**Comment by theacodes** [comment]
Hi @monkeywithacupcake! We'd love to merge this but there's some outstanding comments by @stsewd. Do you have time to address these or should we take on the PR ourselves from here?

**Comment by theacodes** [comment]
I'm going to close this PR due to inactivity. That's not to say I'm not interested in merging it! If you're still up for it just reply and I'll re-open the PR. :)

**Pull request closed without merging**